vscode-antimony/src/annotationInput.ts

```diff
@@ -5,19 +5,70 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { QuickPickItem, window, Disposable, CancellationToken, QuickInputButton, QuickInput, ExtensionContext, QuickInputButtons, Uri, commands, QuickPick } from 'vscode';
-import { LogMessageNotification } from 'vscode-languageclient';
+import { integer, LogMessageNotification } from 'vscode-languageclient';
 import { sleep } from './utils/utils';
 import { ProgressLocation } from 'vscode'
+import { Stats } from 'webpack';
 
 /**
  * A multi-step input using window.createQuickPick() and window.createInputBox().
  * 
  * This first part uses the helper class `MultiStepInput` that wraps the API for the multi-step case.
  */
-export async function multiStepInput(context: ExtensionContext, initialEntity: string = null) {
-    const databases = [
-        { label: 'ChEBI', id: 'chebi' },
-        { label: 'UniProt', id: 'uniprot'}];
+export async function multiStepInput(context: ExtensionContext, initialEntity: string = null, selectedType: string = null) {
+    console.log("3 selected type: " + selectedType);
+    var databases;
+    if (selectedType === 'compartment') {
+        databases = [
+            { label: 'Recommended Databases:'},

```

Comment:
I may be looking at this too soon. I recall that you were going to eliminate this structure for the popup and instead use an "*" at the end of line?